### PR TITLE
Checkout: AB Test PayPal logo instead of just name

### PR DESCRIPTION
**Pull request by yoavf**
AB Test the use of a PayPal logo instead of just the name.

Before:
screen shot 2017-07-19 at 1 32 18

After:
screen shot 2017-07-19 at 1 43 15

TODO:
- [ ] Design review/fix (logo positioning feels off)
- [ ] Limit AB test to specific country/locale

### CODE REVIEW

File: client/my-sites/checkout/checkout/credit-card-payment-box.jsx [hunk 1]
```diff
@@ -173,12 +183,12 @@ var CreditCardPaymentBox = React.createClass( {
...
+				title={ this.props.translate( 'Secure Payment' ) }>
 				{ this.content() }
 			</PaymentBox>
 		);
 	}
 
 } );
 
-module.exports = CreditCardPaymentBox;
+module.exports = localize( CreditCardPaymentBox );
```
Comment (akirk) [inline client/my-sites/checkout/checkout/credit-card-payment-box.jsx]: What's the benefit of this change?

Comment (yoavf) [inline client/my-sites/checkout/checkout/credit-card-payment-box.jsx]: That's enforced by a pre-commit hook - `wpcalypso/i18n-no-this-translate`, see https://git.io/vSwRi . The goal is to move away from mixins iirc.

Comment (akirk) [inline client/my-sites/checkout/checkout/credit-card-payment-box.jsx]: Ah ok. Thanks for the pointer!

**Comment by yoavf** [comment]
Just waiting for the string to drop on WP.com, where I'll manually copy over the existing translations.

**Pull request merged**